modules/ROOT/pages/troubleshooting-visualization.adoc

```diff
@@ -0,0 +1,87 @@
+= Troubleshooting Visualization
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+3
+
+== About the Troubleshooting Visualization
+
+The Troubleshooting Visualization shows you the current and past health of your Mule apps. It allows you to quickly visualize where your network is slow, receiving heavy traffic or failing. From this visualization, you can drill down directly into Anypoint Monitoring dashboards or logs to continue to troubleshoot the performance of your systems.
```

Comment:
GLOBALLY replace "allows you" with "enables you" in all writing.
```suggestion
The Troubleshooting Visualization shows you the current and past health of your Mule apps. It enables you to quickly visualize where your network is slow, receiving heavy traffic, or failing. From this visualization, you can drill down directly into Anypoint Monitoring dashboards or logs to continue to troubleshoot the performance of your systems.
```

Reply:
GLOBALLY evaluate the texts that follow your Level-1 titles to ensure that they create a user expectation for the Level-2 topics...even naming them.